server/src/services/backup.service.ts

```diff
@@ -81,8 +81,22 @@ export class BackupService extends BaseService {
 
     const isUrlConnection = config.connectionType === 'url';
 
+    // Remove known bad url parameters for pg_dumpall (e.g.
+    // `libpqcompat`) so the CLI doesn't fail with messages like
+    // `invalid sslmode value: "no-verify"`.
+    let connectionUrl: string | null = isUrlConnection ? config.url : null;
+    if (connectionUrl !== null) {
+      try {
+        const url = new URL(connectionUrl);
+        url.searchParams.delete('libpqcompat');
+        connectionUrl = url.toString();
+      } catch (e) {
```

Comment:
I think you can assume at this point that the url is valid since Immich wouldn't even have started otherwise. So `new URL()` won't ever throw and you don't need the try/catch